apps/docs/src/.vitepress/components/IconLibraryItem.vue

```diff
@@ -1,6 +1,8 @@
 <script lang="ts" setup>
-import { ref } from "vue";
+import check from "@sit-onyx/icons/check.svg?raw";
```

Comment:
imho circle-check looks nicer but it's fine ^^
<img width="227" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/5151afb7-42c7-4839-8976-1cd25f8c0f29">

`import circleCheck from "@sit-onyx/icons/circle-check.svg?raw";`

Reply:
Done :)